RafRate.API/Configuration/Extentions/ServicesExtentions.cs

```diff
@@ -1,14 +1,15 @@
-﻿using RafRate.Core.Interfaces;
-using RafRate.Core.Services;
+﻿using RafRate.Core.Services;
+using Microsoft.EntityFrameworkCore;
+using RafRate.Data.Contexts;
 
-namespace RafRate.API.Extentions
+namespace RafRate.API.Configuration.Extentions
 {
     public static class ServicesExtentions
     {
         public static IServiceCollection ConfigureServices(this IServiceCollection services)
             => services.AddScoped<CommentService>()
-                       .AddScoped<AuthenticationService>()
-                       .AddScoped<IActiveDayService>()
-                       .AddScoped<TeacherService>();
+                .AddScoped<AuthenticationService>()
+                .AddScoped<ActiveDayService>()
+                .AddScoped<TeacherService>();
     }
```

Comment:
Try to use
.AddScope<InterfaceName, RealisationClassName>()
It will be work correctly